### PR TITLE
[v6r19] add setup.py

### PR DESCRIPTION
**Pull request by chaen**
I would like to add a first version of setup.py.
It contains no dependency, does not distributes the tests, and distributes the scripts with their `py` extension.

BEGINRELEASENOTES
*None
NEW: add dummy setup.py
ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
Did you see #2775 ? Can you compare?

**Comment by chaen** [comment]
yep I saw. But I don't have the same ambition :)
I just did a setup.py that install DIRAC only. No dependencies

**Comment by fstagni** [comment]
Wouldn't we need to change the structure here? I mean, having a DIRAC/DIRAC/*System ?

**Comment by chaen** [comment]
This would be ideal :-) But it is not necessary, I do all the mangling in the setup.py

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# at the root
+allPackages.insert(0, 'DIRAC')
+package_dir['DIRAC'] = base_dir
+
+# The scripts to be distributed
+scripts = glob.glob('%s/*/scripts/*.py' % base_dir)
+
+setup(
+    name="DIRAC",
+    version="6.19.0",
```
Comment (atsareg) [inline setup.py]: This version is supposed to be updated with each patch release ?

**Comment by atsareg** [comment]
Making DIRAC/DIRAC/*System structure should not be a problem. We can make itas part of v6r20 I think

**Pull request merged**